tools/cli/skeletons/packages/src/class-example.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Package description here
+ *
+ * @package automattic/package-name
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class description.
+ */
+class Package_Name {
```

Comment:
The updated generate command works well enough for me. It'd be nice if `Package_Name` was updated with the `package-name` as well, but not a dealbreaker.

Reply:
Nice, was gonna do this for a followup, but was easier than I thought.